### PR TITLE
[WIP] Nodejs Jenkins Agent upgrade release note 4.x

### PR DESCRIPTION
**Pull request by codyhoag**
This is a sample for the release note we'll add for the 4.2-4.4 z-stream releases describing the upcoming additional two Nodejs Jenkins Agent images. This PR is for review purposes only.

Preview: http://file.rdu.redhat.com/~choag/052720/nodejs-image-test-4.x/release_notes/ocp-4-4-release-notes.html#ocp-4-4-x-added-nodejs-jenkins-agent-v10-and-v12

### CODE REVIEW

**Comment by codyhoag** [comment]
@akram can you review this 4.x edition for accuracy? Thanks!

**Comment by codyhoag** [comment]
@sferich888 PTAL? This is intended for 4.2-4.4.

File: release_notes/ocp-4-4-release-notes.adoc [hunk 1]
```diff
@@ -2342,3 +2342,47 @@ After this one-time manual restart, subsequent upgrades and rotations will
...
 intervention.
 ====
+
+[[ocp-4-4-x-features]]
+==== Features
+
+[[ocp-4-4-x-added-nodejs-jenkins-agent-v10-and-v12]]
+===== Added Node.js Jenkins Agent v10 and v12
+
+The `jenkins-agent-nodejs-10-rhel7` and `jenkins-agent-nodejs-12-rhel7` images
```
Comment (ahardin-rh) [inline release_notes/ocp-4-4-release-notes.adoc:2352]: s/have been/are now

File: release_notes/ocp-4-4-release-notes.adoc [hunk 2]
```diff
@@ -2342,3 +2342,47 @@ After this one-time manual restart, subsequent upgrades and rotations will
...
+have been added to {product-title}. These new images allow Jenkins Pipelines to
+be upgraded to use either v10 or v12 of the Node.js Jenkins Agent. The Node.js
+v8 Jenkins Agent has been deprecated, but will continue being provided. For
```
Comment (ahardin-rh) [inline release_notes/ocp-4-4-release-notes.adoc:2355]: s/has been/is now
s/continue being provided/continue to be

**Comment by ahardin-rh** [comment]
@codyhoag Just some minor comments from me. Looks good!

**Comment by codyhoag** [comment]
Thanks all! Docs have been approved and are now ready. :+1: 

These edits will be applied to the 4.y.z release Errata when the new Node.js images are made available. This PR will serve as the review for that future PR.

**Pull request closed without merging**